dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/WorkerGroupServiceImpl.java

```diff
@@ -122,6 +123,7 @@ public Map<String, Object> saveWorkerGroup(User loginUser, int id, String name,
             return result;
         }
         if (workerGroup.getId() != 0) {
+            handleWorkGroup(workerGroup);
```

Comment:
This is strange, I don't think we should add a handle method here, what the handle method means?

Reply:
Persistent equal burial point